README_DEVELOPERS.md

```diff
@@ -8,6 +8,11 @@ We use the [Fork-and-Branch workflow](http://blog.scottlowe.org/2015/01/27/using
 
 Make sure that you properly configure your Git [username](https://help.github.com/articles/setting-your-username-in-git) and [email](https://help.github.com/articles/setting-your-email-in-git).
 
+* [Install Required Software](#install-required-software)
```

Comment:
I don't like this list.
It doesn't really look like a TOC.

Reply:
I can remove it if you insist.

Reply:
Yes please.
This change is out of the PR *scope* 😏